flarestack/data/icecube/northern_tracks/nt_v005_p01.py

```diff
@@ -0,0 +1,65 @@
+from flarestack.data.icecube.ic_season import IceCubeDataset, get_dataset_dir
+from flarestack.data.icecube.northern_tracks import (
+    NTSeasonNewStyle,
+    get_diffuse_binning,
+)
+
+icecube_dataset_dir = get_dataset_dir()
```

Comment:
Is this the way the code needs to run now? i.e you need imports at the top, then `get_dataset_dir()`, then `import dataset_index`

The pythonic way is usually for all imports to go together at the top

Reply:
This was unintended, indeed.